ontology/yaml/resources/units/units.yaml

```diff
@@ -32,27 +45,83 @@ density:
 - grams_per_cubic_meter
 - micrograms_per_cubic_meter
 - milligrams_per_cubic_meter
+electriccapacitance:
+- farads: STANDARD
+electricconductance:
+- siemens: STANDARD
+electricconductivity:
+- siemens_per_meter: STANDARD
+electriccurrentdensity:
+- amperes_per_square_meter: STANDARD
+electricfield:
+- volts_per_meter: STANDARD
+electricinductance:
+- henrys: STANDARD
+electricphasedifference:
+- degrees_phase: STANDARD
+electricfieldstrength:
+- amperes_per_meter: STANDARD
+electricresistivity:
+- ohm_meters: STANDARD
 energy:
 - joules: STANDARD
 - kilowatt_hours
+- btus
+- kilobtus
+- kilojoules
+- megabtus
+- megajoules
 - megawatt_hours
+- therms
+- ton_hours
+- watt_hours
+energydensity:
+- kilojoules_per_kilogram: STANDARD
+entropy:
+- joules_per_degree_kelvin: STANDARD
+- kilojoules_per_degree_kelvin
+- megajoules_per_degree_kelvin
 flowrate:
 - cubic_meters_per_second: STANDARD
-- cubic_feet_per_minute
 - cubic_meters_per_minute
 - cubic_meters_per_hour
-- gallons_per_minute
 - liters_per_second
-- kilograms_per_hour
+- liters_per_minute
+- liters_per_hour
+- cubic_feet_per_second
+- cubic_feet_per_minute
+- gallons_per_minute
+- imperial_gallons_per_minute
+- us_gallons_per_minute
+force:
+- newton: STANDARD
 frequency:
 - hertz: STANDARD
+- cycles_per_hour
+- cycles_per_minute
+- kilohertz
+- megahertz
+- per_hour
+- per_minute
+- per_second
+- percent_per_second
 humidity:
 - percent_relative_humidity: STANDARD
+- grams_of_water_per_kilogram_dry_air
```

Comment:
we model humidity as a percentage, not as this. we cover absolute humidity differently, since it can either be measured in units of mass/mass or mass/volume. see here: https://github.com/google/digitalbuildings/blob/407075cbfbd3d81e6c2ce0c25cc160e3238d9225/ontology/yaml/resources/fields/telemetry_fields.yaml#L999

Reply:
They are both adimensional units.
Maybe we should rename "humidity" to "relative humidity" and then add "absolute humidity"?

Reply:
Relative and absolute humidity are not both dimensionless. 

Relative humidity is a percentage, which is dimensionless. 

Absolute humidity is can be dimensionless (g water/kg dry air), but it can be represented in ways that have units (g water/m^3 dry air).

I think the easiest thing to do here is add `grams_of_water_per_kilogram` as units under concentration, and then we can use the fields from lines 999/1000 i linked above.

Reply:
I've decided to remove this unit for now.